dashboard.py

```diff
@@ -83,7 +90,7 @@ def parse_contents(contents, filename):
         # will likely include categorical options in later instance (sooner)
     processed_df, cat_list = get_data(df, mapping)
     all_species = get_species_options(processed_df)
-    
+    print(mapping)
```

Comment:
Did you mean to leave this print in?

Reply:
No, I thought I had removed all of those. Thanks for catching that.